libs/utils/src/auth.ts

```diff
@@ -6,6 +6,12 @@ export function isSystemAdministratorAuth(
 export function isSystemAdministratorAuth(
   auth: InsertedSmartCardAuth.AuthStatus
 ): auth is InsertedSmartCardAuth.SystemAdministratorLoggedIn;
+export function isSystemAdministratorAuth(
+  auth: DippedSmartCardAuth.AuthStatus | InsertedSmartCardAuth.AuthStatus
+): auth is
+  | InsertedSmartCardAuth.SystemAdministratorLoggedIn
+  | DippedSmartCardAuth.SystemAdministratorLoggedIn;
+
```

Comment:
This adds an [overload signature](https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads) for the function overload that accepts the union type of either `AuthStatus`